src/utility/Validation.js

```diff
@@ -71,6 +72,13 @@ const validateTrueFunction = thing => {
   }
 }
 
+const mandatoryURLFunction = thing => {
+  return {
+    error: thing ? true : !validateURL(thing),
```

Comment:
Sorry having trouble wrapping my head around line 77 - so `thing` here is the URL, and if it exists at all it will return saying that an error exists?

Reply:
Thank you for catching that! I've definitely swapped the two terms in the ternary statement, it should validate the URL when there is a thing and set error to true when the input is empty (will add empty string comparison as well)